papaparse.js

```diff
@@ -483,10 +482,10 @@
 					finished: finishedIncludingPreview
 				});
 			}
-			else if (isFunction(this._config.chunk))
+			else if (isFunction(this._config.chunk) && !isFakeChunk)
 			{
 				this._config.chunk(results, this._handle);
-				if (this._paused)
+				if (this._handle.paused() || this._handle.aborted())
```

Comment:
`|| this._handle.aborted()` is the one addition to this PR from #166. It's necessary to avoid calling `complete` twice when `abort` is called.